spa/plugins/bluez5/bluez5-monitor.c

```diff
@@ -1227,6 +1227,18 @@ static int register_a2dp_endpoint(struct spa_bt_monitor *monitor,
 			return -ENOTSUP;
 		}
 		break;
+	case SPA_BT_PROFILE_A2DP_SINK:
+		switch (codec) {
+		case A2DP_CODEC_SBC:
+			profile_path = "/A2DP/SBC/Sink";
+			break;
+		case A2DP_CODEC_MPEG24:
```

Comment:
We should probably leave the MPEG24 code paths commented out, since we don't really support it (the phone should fall back to SBC if we don't advertise MPEG24, but if we advertise MPEG24 and the phone decides to use it, nothing will work)